pkgs/development/libraries/sqlite/tools.nix

```diff
@@ -4,11 +4,11 @@ let
   archiveVersion = import ./archive-version.nix stdenv.lib;
   mkTool = { pname, makeTarget, description, homepage }: stdenv.mkDerivation rec {
     inherit pname;
-    version = "3.34.0";
+    version = "3.34.1";
 
     src = assert version == sqlite.version; fetchurl {
-      url = "https://sqlite.org/2020/sqlite-src-${archiveVersion version}.zip";
-      sha256 = "0giklai05shqalj1wwadi9hg5dx6vff8nrblqh9xxljnrq701hm5";
+      url = "https://sqlite.org/2021/sqlite-src-${archiveVersion version}.zip";
+      sha256 = "0giklai05shqalj1wwadi9hg5dx6vff8nrblqh9xxljnrq701h00";
```

Comment:
Something isn't right here:

```
hash mismatch in fixed-output derivation '/nix/store/xx4h0r6dd9wr842q2ccaz70p77xyjb29-sqlite-src-3340100.zip.drv':
  specified: sha256-AMAADs5W0t4TxHRli5zbprfyYIpNcR4kVRjqAqKiMz4=
     got:    sha256-3d0jeZawlt7os3FGx6N6YmqAMG1mlRA9LsFu47hS/0k=
```

See: https://hydra.nixos.org/job/nixpkgs/trunk/sqldiff.x86_64-linux

Reply:
🤦 Thanks for catching this, looks like I didn't actually update the checksum.

Fixed in https://github.com/NixOS/nixpkgs/pull/114441